sftp.c

```diff
@@ -1778,6 +1797,20 @@ parse_dispatch_command(struct sftp_conn *conn, const char *cmd, char **pwd,
 			err = 1;
 		}
 		break;
+	case I_CLR_RCPT:
+		iron_reset_recipients();
+		break;
+	case I_SHOW_RCPT:
+		iron_list_recipients();
```

Comment:
The command isn't `lsrcpt` because all commands in sftp that start with `l` are "local" commands (e.g. `lpwd`, `lis`, `lcd`).
I'll change the function names.